<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lobby</class>
 <widget class="QDialog" name="Lobby">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Lobby</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NewNameLobby">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>231</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AddLobby">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>350</y>
     <width>51</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>350</y>
     <width>321</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Join</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of players</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="NewLobbyNumberPlayers">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>390</y>
     <width>48</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Lobbies</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>50</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Players</string>
   </property>
  </widget>
  <widget class="QListWidget" name="ListLobbyW">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>301</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="ListPlayersW">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>321</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>390</y>
     <width>321</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refresh">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>refresh</string>
   </property>
  </widget>
  <widget class="QPushButton" name="show">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>280</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>show</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
